src/syscheckd/cppcheckSuppress.txt

```diff
@@ -2,3 +2,4 @@
 *:*src/syscheckd/src/db/src/file.cpp:188
 *:*src/syscheckd/src/db/src/db.cpp:160
 *:*src/syscheckd/src/db/src/db.cpp:119
+*:*src/syscheckd/src/create_db.c:712
```

Comment:
this ignore should be in line 713 `txn_context->latest_entry = &new_entry;` because RTR will fail with this supress